parquet/src/main/java/org/apache/iceberg/parquet/ParquetUtil.java

```diff
@@ -78,8 +80,14 @@ public static Metrics footerMetrics(ParquetMetadata metadata) {
       rowCount += block.getRowCount();
       for (ColumnChunkMetaData column : block.getColumns()) {
         ColumnPath path = column.getPath();
-        int fieldId = fileSchema.aliasToId(path.toDotString());
+        String columnAlias = path.toDotString();
+        int fieldId = fileSchema.aliasToId(columnAlias);
         increment(columnSizes, fieldId, column.getTotalSize());
+
+        MetricsMode columnMetricsMode = metricsSpec.getColumnMetricsMode(columnAlias);
```

Comment:
The column alias should be Iceberg's column alias, not Parquet's column alias. So this should look up the name from the Iceberg schema before checking the spec.